jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/events/WorkflowBridgeTaskLifeCycleEventListener.java

```diff
@@ -39,20 +39,23 @@ public WorkflowBridgeTaskLifeCycleEventListener(RuntimeManager runtimeManager) {
 
     @Override
     public void afterTaskSuspendedEvent(TaskEvent event) {
-        executeWork(event.getTask(), HumanTaskNodeInstance.SUSPEND_SIGNAL);
+        String suspendUntilExpression = (String) event.getMetadata().get("suspendUntil");
```

Comment:
Perhaps it's more clear to define a constant for "suspendUntil" as it's used several times in this class.

Reply:
create constant en HumanTaskNodeInstance